components/form/input.scss

```diff
@@ -1,36 +1,21 @@
-$form-input-padding: $form-input-padding-left 0.9rem;
+$form-input-padding: $form-input-padding-left;
 $form-input-border-radius: $border-radius;
-$form-input-border: 1px solid var(--hdd-color-grey);
+$form-input-border: 1px solid var(--hdd-color-grey-border);
 
 .hdd-form_input {
   // Variable containing '&' to jump to current parent inside a child class.
   $this: &;
 
   display: flex;
   box-sizing: border-box;
-
   align-items: center;
-  padding: $form-input-padding;
+  padding: var(--hdd-spacing-1);
+  padding-left: $form-input-padding;
   transition: border-color $medium-transition $transition-cubic;
   border: $form-input-border;
   border-radius: $form-input-border-radius;
-
   background: var(--hdd-color-white);
-
-  &:not(.is-invalid):focus-within {
-    @extend .hdd-form_input, .is-focus;
-  }
-
-  &.is-focus {
-    border-color: var(--hdd-color-primary);
-
-    #{$this}_icon,
-    svg,
-    i {
-      opacity: 1;
-      color: var(--hdd-color-primary);
-    }
-  }
+  line-height: 150%;
```

Comment:
hvorfor?

Reply:
Er definget med dette i design documentet

Reply:
Stylen i firgma wer auto generert og inneholder ofte en del rar eller unødvendig css. Fjern alt som ikke har noen effekt

Reply:
Den er god, men da venter jeg også bare til typographien er ferdig før denne branchen gjøres noe mer med. Så slipper vi to omganger med endringer